model/ProctorioApiService.php

```diff
@@ -151,4 +138,25 @@ private function getProctorioLibraryService(): ProctorioService
 
         return $this->proctorioService;
     }
+
+    public function findReviewUrl(string $deliveryId): ?string
```

Comment:
```suggestion
    public function findReviewUrl(string $deliveryId): string
```
This is always returning a string cause of the casting